controllers/comments.controllers.js

```diff
@@ -7,4 +7,15 @@ exports.getCommentsByArticleID = (req, res, next) => {
         res.status(200).send({comments})
     })
     .catch(next)
+}
+
+exports.postCommentByArticleID = (req, res, next) => {
+    const {article_id} = req.params
+    const {username, body} = req.body
+    const comment = [0, new Date(), username, body, article_id]
```

Comment:
If you check the seed, votes and created_at have defaults set so you don't need to create them or include them in your insert in the model